src/pages/KaKaoLoginPage/KaKaoLoginPage.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useKaKaoLogin } from '@/hooks/useAuth';
+
+const KaKaoLoginPage = () => {
+  const navigate = useNavigate();
+
+  const AUTHORIZATION_CODE: string = new URL(
+    document.location.toString()
+  ).searchParams.get('code') as string;
+
+  const { mutate: mutateKaKaoLogin } = useKaKaoLogin({
+    onSuccess: () => {
+      alert('로그인 성공');
+      navigate('/main');
```

Comment:
뒤로가기를 막는 [replace](https://stackoverflow.com/questions/72794430/what-does-usenavigate-replace-option-do):true를 추가해도 좋을 것 같아요!! 👍 
```suggestion
      navigate('/main',{replace:true);
```

Reply:
`replace: true` 추가하겠습니다!